### PR TITLE
[Feature] Add Image

### PR DESCRIPTION
**Pull request by sk97kang**
## Description

Add Image and ImageStyle

## Related Issues

#4 

## Tests

X

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [X] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [X] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [X] Run `yarn lint`
- [X] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: main/Card/index.tsx [hunk 1]
```diff
@@ -17,11 +23,27 @@ interface Props {
...
-  return <Container style={[containerStyle]}>{children}</Container>;
+  const { containerStyle, children, image, imageStyle } = props;
+  return (
+    <Container style={[containerStyle]}>
+      {image && (
+        <Image
+          source={image}
+          style={{
+            width: 100,
+            height: 100,
```
Comment (greatSumini) [inline main/Card/index.tsx]: recommend to replace with styled-component

File: main/Card/index.tsx [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
+import {
+  ViewStyle,
+  Image,
+  ImageSourcePropType,
+  StyleProp,
+  ImageStyle,
+} from 'react-native';
 
 import styled from 'styled-components/native';
+import { ImageProps } from 'react-native-svg';
```
Comment (greatSumini) [inline main/Card/index.tsx]: It doesn't seem to be used. 🧐

File: main/Card/index.tsx [hunk 3]
```diff
@@ -17,11 +23,27 @@ interface Props {
...
+        <Image
+          source={image}
+          style={{
+            width: 100,
+            height: 100,
+            ...imageStyle,
+          }}
+        />
+      )}
+      {children}
```
Comment (greatSumini) [inline main/Card/index.tsx]: need ContentContainer(padding: 12px 16px)

File: main/Card/index.tsx (hunk 1 above)
Comment (sk97kang) [reply main/Card/index.tsx]: I use styled-component.

File: main/Card/index.tsx (hunk 2 above)
Comment (sk97kang) [reply main/Card/index.tsx]: I erased it.

File: main/Card/index.tsx (hunk 3 above)
Comment (sk97kang) [reply main/Card/index.tsx]: I think we need a container for children.

Comment (pickk-dev) [reply main/Card/index.tsx]: i agree. could u add that?

File: main/Card/index.tsx [hunk 4]
```diff
@@ -19,6 +12,11 @@ const Container = styled.View`
   shadow-offset: 0px 2px;
 `;
 
+const ImageContainer = styled.Image`
```
Comment (pickk-dev) [inline main/Card/index.tsx]: i think it's better to rename it to StyledImage

Comment (sk97kang) [reply main/Card/index.tsx]: I agree. I changed its name.

File: main/Card/index.tsx (hunk 3 above)
Comment (sk97kang) [reply main/Card/index.tsx]: Ok. I Added ChildrenContainer

**Pull request merged**